tests/pyreverse/test_diadefs.py

```diff
@@ -22,25 +22,30 @@
 # pylint: disable=redefined-outer-name
 import sys
 from pathlib import Path
+from typing import Callable, Dict, List, Tuple
 
 import pytest
 from astroid import nodes
+from conftest import PyreverseConfig  # type: ignore #pylint: disable=no-name-in-module
```

Comment:
I think we can remove this one and type it properly later.

Reply:
I would be in favour of letting this in.
We will need to write these lines at some point anyway and the disabled messages serve as a reminder that something needs to be fixed. Removing the typing makes it less obvious that there is still something on the "to-do list".

But obviously, this is up to you if you still want me to remove it from the typing let me know!

Reply:
No I think you're right. I was wondering why the class definition is in conftest which is probably not something we should explore in this MR.